python/grass/grassdb/history.py

```diff
@@ -0,0 +1,88 @@
+"""
+Managing existing history files included within mapset
+
+(C) 2023 by the GRASS Development Team
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+
+.. sectionauthor:: Linda Karlovska (Kladivova) linda.karlovska@seznam.cz
+"""
+
+import os
+import shutil
+import codecs
```

Comment:
codecs.open should be replaced with simple open, but include encoding for pylint: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/unspecified-encoding.html